packages/react-sdk/src/embed-template.test.ts

```diff
@@ -384,3 +381,86 @@ test("generate data for embedding from props bound to data source expressions",
     styles: [],
   });
 });
+
+test("generate data for embedding from action props", () => {
+  expect(
+    generateDataFromEmbedTemplate(
+      [
+        {
+          type: "instance",
+          component: "Box1",
+          props: [
+            {
+              type: "string",
+              name: "state",
+              value: "initial",
+              dataSourceRef: { type: "variable", name: "boxState" },
+            },
+          ],
+          children: [
+            {
+              type: "instance",
+              component: "Box2",
+              props: [
+                {
+                  type: "action",
+                  name: "onClick",
+                  value: [{ type: "execute", code: `boxState = 'success'` }],
```

Comment:
code vs expression, can it be more than expression? If yes then code makes sense